packages/playwright-core/src/server/chromium/crNetworkManager.ts

```diff
@@ -241,7 +250,7 @@ export class CRNetworkManager {
       return;
     }
 
-    if (!frame) {
+    if (!frame && !this._serviceWorker) {
```

Comment:
Let's add a comment that non-service-worker requests must have a frame. If we didn't find one, pretend there was not request :)